### ISSUE TITLE
Could Seq2SeqSharp contribute to Tensorflow.NET NLU features in win-win way? 

### ISSUE THREAD

**Issue by GeorgeS2019** [zhongkaifu/Seq2SeqSharp#19]
Suggestions
- Currently Tensorlfow.NET lacks key [Keras layers](https://keras.io/api/layers/) relevant for NLU (e.g. seq2seq and transformer). 
For examples:
  -  [Normalization layers](https://keras.io/api/layers/normalization_layers/)
     -  LayerNormalization
  -  [Attention layers](https://keras.io/api/layers/attention_layers/)
     -  MultiHeadAttention
     -  Attention 

LayerNormalization, MultiHeadAttention and Attention layers in C# (for example) have been implemented in Seq2SeqSharp

The questions:
- Could these Seq2SeqSharp codes useful to speed up the implementation of NLU Keras layers in Tensorflow.NET?
- Is there a need for Seq2SeqSharp to adopt Keras layer naming? 
   - This could help codes implemented in Tensorflow.NET to be relevant for Seq2SeqSharp and vice versa. 
   - This also make more developers aware of the transformer framework in Seq2SeqSharp.
   - Hopefully more .NET developers could join and speed up transformer NLU development for the .NET community.
